flask_resources/errors.py

```diff
@@ -75,47 +117,64 @@ def get_body(self, environ=None):
         return body
 
 
-# FIXME: From here down need review
-
+class SchemaValidationError(RESTException):
+    """Marshmallow schema validation exception."""
```

Comment:
Minor: Data validation is not tight to Marshamallow. We do use it, but the code in `invenio-records-resources` allows the usage of any custom validation.
```suggestion
    """Schema validation exception."""
```

In addition, since data validation is a concept introduced in `invenio-records-resource` shouldn't this exception be defined there? Not in the service errors, but in the global (resources) errors.

Reply:
I agree :) I didn't really focus on that and definitely it would require some more iteration down the road :)

Reply:
If you open an issue about this so we don't forget, PR is good to go :)

Reply:
https://github.com/inveniosoftware/flask-resources/issues/69 done :)